main.go

```diff
@@ -208,6 +208,12 @@ func main() {
 		1000000,
 		"number of bytes a file can contain before being removed from output",
 	)
+	flags.StringVar(
+		&processor.CountAs,
+		"count-as",
+		"",
+		"count extension as language [e.g. jsp,htm:chead,\"C Header\" maps extension jsp to html and chead to C Header]",
```

Comment:
Purely personal preference, but using `:` to separate key/value and `,` to separate the pairs seems more "normal" to me. My brain keeps wanting to associate `htm` with `chead` in that example.

i.e. `jsp:htm,chead:"C Header"`

Reply:
Yeah I was not sure which way to go with it. Ill flip it around. I don't have a strong preference either way.

Reply:
My actual preference would be to have one flag per re-mapping (e.g. `--count-as jsp:htm --count-as chead:"C Header"`), but that seems rather awkward to do with just the basic `flags` package.

Reply:
Yeah one of those annoying things. Something to make as a breaking change in the future perhaps.